build.gradle.kts

```diff
@@ -9,13 +9,26 @@ configurations {
 
 dependencies {
   // Spring boot dependencies
+  implementation("org.springframework.boot:spring-boot-starter-security")
   implementation("org.springframework.boot:spring-boot-starter-webflux")
   implementation("uk.gov.justice.service.hmpps:hmpps-kotlin-spring-boot-starter:0.2.4")
+  implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+
+  // Database dependencies
+  runtimeOnly("org.flywaydb:flyway-core")
+  runtimeOnly("org.postgresql:postgresql:42.7.1")
 
   // Test dependencies
   testImplementation("io.jsonwebtoken:jjwt-impl:0.12.5")
   testImplementation("io.jsonwebtoken:jjwt-jackson:0.12.5")
   testImplementation("org.wiremock:wiremock-standalone:3.5.3")
+  testImplementation("com.h2database:h2")
```

Comment:
OK for now to use H2 but in this project but ideally I'd really prefer to start using TestContainers.

If we do it from the outset shouldn't be too much bother.

Reply:
Happy with a move to testcontainers so we get like-for-like postrgresql during tests, but ok for this merge I think. Will be a separate ticket to add TC pgsql.